test/extensions/http/header_validators/envoy_default/path_normalizer_corpus/corpus-1

```diff
@@ -0,0 +1,9 @@
+method: "GET"
```

Comment:
Nice job naming the other files, they really capture the test case. Can this have a similar descriptive name?